package.json

```diff
@@ -43,5 +46,10 @@
     "url": "https://github.com/BrianMacIntosh/alexa-skill-test-framework/issues",
     "email": "brianamacintosh@gmail.com"
   },
-  "repository": "BrianMacIntosh/alexa-skill-test-framework"
+  "repository": "BrianMacIntosh/alexa-skill-test-framework",
+  "devDependencies": {
+    "alexa-sdk": "^1.0.10",
```

Comment:
This is needed to run the tests in the examples folder. Do you see problems with this?